test/kafka-exactly-once/mzcompose.yml

```diff
@@ -0,0 +1,116 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+version: '3.7'
+
+mzworkflows:
+  kafka-exactly-once:
+    env:
+      KAFKA_VERSION: latest
+      # @Philip, how could I get a random seed in here?
```

Comment:
@philip-stoev How could I get a random nonce that is stable for the whole workflow? We need this here because we have multiple testdrive invocations and the nonce/seed is appended to Kafka topics.

Reply:
Can you just use --seed 1 ? Why does it need to be random?

Reply:
Making it random reduces the likelihood of clashes, i.e. if I repeatedly run the test on my machine. But could be that it's not an issue.